### ISSUE TITLE
iOS app not opening on MacOS after installed Proxyman

### ISSUE THREAD

**Issue by davidnilsson-abillion** [ProxymanApp/Proxyman#1475]
## Description
I installed Proxyman and now can't start an iOS app on M1 Macos.

## Steps to Reproduce

1. Started iOS app fine on M1 Macos
2. Installed Proxyman, installed cert
3. Starting iOS app failed with error msg, see attached screenshot

![image](https://user-images.githubusercontent.com/88371040/209301696-99b30c5d-bf42-4178-9d21-0a6d4942e861.png)

## Environment

- App version: e.g Proxyman 3.9.0
- macOS version: e.g macOS Monterey

**Comment by davidnilsson-abillion** [comment]
BTW. The iOS app I'm trying to start is installed on Macos using testflight.
It worked fine before installing Proxyman

**Comment by NghiaTranUIT** [comment]
If you quit Proxyman, does your iOS (on M1 Mac) works?

If should would since Proxyman is closed and it does nothing with the system.

**Comment by davidnilsson-abillion** [comment]
Yes it does, but I was hoping to be able to inspect the traffic from the iOS app I run on the Macbook.

Best,
David

**Comment by NghiaTranUIT** [comment]
Can you share with me the name of the iOS app? If it's a popular app, such as IG, Facebook, or Whatsapp, there is a high chance that it's protected by SSL Pinning, which prevents the Man-in-the-middle apps to intercept their HTTPS traffic.

Thus, you will get SSL Error.

**Comment by davidnilsson-abillion** [comment]
It’s our own app, I’m installing it using testflight.

The error I get is “File not found” as I shared in a screenshot.

**Comment by NghiaTranUIT** [comment]
Not sure how to fix it. If you open your iOS app on M1 Macbook (not by using Testflight), does it work?

I guess Testflight might call some traffic, which is failed when using Proxyman.

**Comment by davidnilsson-abillion** [comment]
Hmm… I tried installing the non-testflight, app store version and it works. Then I tried the testflight version again, and that also worked. Weird!?

I have upgraded macos since last time, if that has anything to do with it. Weird.

Anyway, seems to work now! Thanks for the help!

Best,
David
